examples/worst_case_attack_example.py

```diff
@@ -50,6 +51,23 @@
 test_preds = target_model.predict_proba(test_X)
 
 # [TRE] Define some attack parameters
+# Use of configuration file to load parameters
```

Comment:
Why comment it all out? Better to have multiple examples within the script that all run I think? These scripts also work as regression tests so best that they run.

Reply:
Updated script to run all examples run.